src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresDatatypeIT.java

```diff
@@ -133,14 +236,37 @@ private void createTableWithUuidColumns() throws SQLException {
     log.info("Created table {} with UUID column", tableName);
   }
 
-  private void startTask() {
-    task = new JdbcSinkTask();
-    task.start(props);
+  private void createTableWithLessFields() throws SQLException {
+    log.info("Creating table {} with less fields", tableName);
+    try (Connection c = pg.getEmbeddedPostgres().getPostgresDatabase().getConnection()) {
+      c.setAutoCommit(false);
+      try (Statement s = c.createStatement()) {
+        String sql = String.format(
+            "CREATE TABLE %s(firstName TEXT, jsonid json, userid UUID)",
+            tableName
+        );
+        log.info("Executing statement: {}", sql);
+        s.execute(sql);
+        c.commit();
+      }
+    }
+    log.info("Created table {} with less fields", tableName);
   }
 
-  public void stopTask() {
-    if (task != null) {
-      task.stop();
+  private void createTableWithPrimaryKey() throws SQLException {
+    log.info("Creating table {} with a primary key", tableName);
+    try (Connection c = pg.getEmbeddedPostgres().getPostgresDatabase().getConnection()) {
+      c.setAutoCommit(false);
+      try (Statement s = c.createStatement()) {
+        String sql = String.format(
+            "CREATE TABLE %s(firstName TEXT PRIMARY KEY, lastName TEXT)",
+            tableName
+        );
+        log.info("Executing statement: {}", sql);
+        s.execute(sql);
+        c.commit();
+      }
     }
+    log.info("Created table {} with a primary key", tableName);
```

Comment:
Might be worth refactoring this into a generic `createTable` or even just `executeSql` method that accepts a `String sqlStatement` and executes it. Then `createTableWithUuidColumns`, `createTableWithLessFields`, and `createTableWithPrimaryKey` could all be reduced to three lines: an initial log statement ("Creating table {} with ..."), a call to the new method, and a final log statement ("Created table {} with ...").